Ports/.port_include.sh

```diff
@@ -18,6 +18,11 @@ buildstep() {
     else
         "$@"
     fi 2>&1 | sed $'s|^|\x1b[34m['"${port}/${buildstep_name}"$']\x1b[39m |'
+    local return_code=${PIPESTATUS[0]}
+    if [ ${return_code} != 0 ]; then
+        echo -e "\x1b[1;31mError in step ${port}/${buildstep_name} (status=${return_code})"
```

Comment:
Can you reset the foreground color back to its default (`\x1b[39m`) after printing the error message? Otherwise this might affect output for other commands:

<img width="1033" alt="Screenshot 2022-10-25 at 08 39 24" src="https://user-images.githubusercontent.com/388571/197700658-a7e7ed80-ad90-4dfc-89a1-9fa064aaf8d4.png">

Reply:
done